apis/Google.Cloud.Firestore/Google.Cloud.Firestore.Tests/SerializationTestData.cs

```diff
@@ -128,19 +128,28 @@ internal static class SerializationTestData
             { new List<ByteString> { ByteString.CopyFromUtf8("abc"), ByteString.CopyFromUtf8("def") },
                 new Value { ArrayValue = new ArrayValue { Values = { new Value { BytesValue = ByteString.CopyFromUtf8("abc") }, new Value { BytesValue = ByteString.CopyFromUtf8("def") } } } } },
 
-            // Map values (that can be deserialized again): dictionaries, attributed types, expandos (which are just dictionaries)
+            // Map values (that can be deserialized again): dictionaries, attributed types, expandos (which are just dictionaries), custom serialized map-like values
+
+            // Dictionaries
             { new Dictionary<string, object> { { "name", "Jon" }, { "score", 10L } },
                 new Value { MapValue = new MapValue { Fields = { { "name", new Value { StringValue = "Jon" } }, { "score", new Value { IntegerValue = 10L } } } } } },
             { new Dictionary<string, int> { { "A", 10 }, { "B", 20 } },
                 new Value { MapValue = new MapValue { Fields = { { "A", new Value { IntegerValue = 10L } }, { "B", new Value { IntegerValue = 20L } } } } } },
+            // Attributed type (each property has an attributed)
```

Comment:
nit: has an attribute (without the d in the end)

Reply:
Will fix, thanks.